tests/test_urldispatch.py

```diff
@@ -1054,14 +1150,14 @@ def test_frozen_router_subapp(app, loop):
     subapp = web.Application()
     subapp.freeze()
     with pytest.raises(RuntimeError):
-        app.add_subapp('/', subapp)
+        app.add_subapp('/pre', subapp)
 
 
 def test_frozen_app_on_subapp(app, loop):
     app.freeze()
     subapp = web.Application()
     with pytest.raises(RuntimeError):
-        app.add_subapp('/', subapp)
+        app.add_subapp('/pre', subapp)
```

Comment:
Is the change required? Why?

Reply:
to check subapp rather than the prefix

    raise ValueError("Prefix cannot be empty")

Reply:
ok